src/server/services/procedures/wildcam/wildcam.js

```diff
@@ -0,0 +1,139 @@
+/**
+ * Wildcam provides access to wildlife images from around the world.
+ * 
+ * Current data sources:
+ * 
+ * - `Zooniverse <https://classroom.zooniverse.org/#/wildcam-gorongosa-lab/explorers/map/>`__
+ * 
+ * @service
+ * @category GeoData
+ * @category Media
+ */
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+const axios = require('axios');
+const logger = require('../utils/logger')('wildcam');
+
+// -------------------------------------------------------------------------
+
+const CSV_PATH = path.join(__dirname, 'wildcam-17nov2022.csv');
+
+// -------------------------------------------------------------------------
+
+const IMG_CACHE = {};
+
+const Wildcam = {};
```

Comment:
It looks like it is WildCam on Zooniverse (rather than Wildcam). It might be nice to use the same convention as Zooniverse does.